website/docs/addon_maya_artist.md

```diff
@@ -39,7 +39,7 @@ between studios and projects but by default your geometry has to be named proper
 For example `sphere_GEO` or `cube1_GEO`. Geometry needs to have freezed transformations
 and must reside under one group, for example `model_GRP`.
 
-![Model example](assets/maya-model_hierarchy_example.jpg)
+![Model example](assets/maya/artist/maya-model_hierarchy_example.jpg)
```

Comment:
if we have a subfolder of `maya`, should we remove `maya` from the file name?

Reply:
That's fair!